### ISSUE TITLE
Trying to run ColabFold v1.5.2

### ISSUE THREAD

**Issue by msafranchik1994** [sokrypton/ColabFold#635]
I'm trying to run ColabFold v1.5.2 but I'm not able. 
I'm using this website: https://colab.research.google.com/github/sokrypton/ColabFold/blob/v1.5.2/AlphaFold2.ipynb#scrollTo=iccGdbe_Pmt9

**Comment by YoshitakaMo** [comment]
How about using the latest one, https://colab.research.google.com/github/sokrypton/ColabFold/blob/main/AlphaFold2.ipynb ?

**Comment by msafranchik1994** [comment]
Yes, I've been recently using the latest one, but I cannot reproduce results that were obtained last year (v1.5.2). Now I wanted to re-run to see if it's a version issue. Thank you very much
